File: packages/typekit-i18n/tests/fixtures/consumer-swift/SmokeApp.swift
import Foundation

@main
struct SmokeApp {
  static func main() throws {
    let bridge = ClosureTranslationRuntimeBridge { key, language, placeholders in
      return "\(key):\(language):\(placeholders.count)"
    }
    let translator = TypekitTranslator(bridge: bridge)
    let value = try translator.translate(
      .title,
      language: .de,
      placeholders: [TranslationPlaceholder(key: "name", value: .string("Ada"))]
    )
    print(value)
  }
}

File: packages/typekit-i18n/tests/fixtures/consumer-kotlin/SmokeApp.kt
fun main() {
  val bridge = LambdaTranslationRuntimeBridge { key, language, placeholders ->
    "$key:$language:${placeholders.size}"
  }

  val translator = TypekitTranslator(bridge = bridge)
  val value = translator.translate(
    key = TranslationKey.TITLE,
    language = TranslationLanguage.DE,
    placeholders = listOf(TranslationPlaceholder("name", TranslationPlaceholderValue.Text("Ada")))
  )

  print(value)
}
